app/assets/stylesheets/config/_import.scss

```diff
@@ -154,7 +156,7 @@ $z-indexes: (
   @each $name, $value in $values {
     .#{$prefix}#{$separator}#{$name} {
       @each $property in $properties {
-        #{$property}: $value;
+        #{$property}: $value !important;
```

Comment:
Related: https://forem-team.slack.com/archives/C01K9U7UATV/p1642687819005100
cc @nickytonline @Ridhwana

Reply:
Bootstrap has entered the chat. 😎

Since this will disappear once we've removed Bootstrap, can you add a TODO comment along those lines?

Reply:
Good suggestion!